src/Microsoft.DotNet.GenAPI/Shared/AssemblySymbolLoader.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace Microsoft.DotNet.GenAPI.Shared
+{
+    internal class AssemblySymbolLoader : IAssemblySymbolLoader
+    {
+        public IAssemblySymbol? LoadAssembly(string path)
+        {
+            using (var stream = File.OpenRead(path))
+            {
+                return LoadAssembly(stream);
+            }
```

Comment:
FYI, this is also possible:
```suggestion
            using var stream = File.OpenRead(path);
            return LoadAssembly(stream);
```